packages/stream_chat_flutter/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Performance improvements
 - Added pinMessage ui support
 - Added `MessageListView.threadSeparatorBuilder` property
+
```

Comment:
Consider adding something about the reaction builder in the Changelog.

Reply:
we're not publishing a release